# Message that informs if the robots of a team are blocked.
#

std_msgs/Header header

# True if the robot is blocked. One for each robot.
bool[3] blocked
